### ISSUE TITLE
Staging: Setting up a program that finds out the stock.

### ISSUE THREAD

**Issue by tort-oise** [tort-oise/stockbot#1]
There are a few options to go about this.

1. Grab it from the Blox Fruits wiki. (slightly unreliable)
2. Grab it from the game through a dev-included script (very unlikely)
3. Grab it from some Discord server, example being Axiore's Discord. (unreliable)

All of these methods need to be implemented dynamically, or in real-time.

Example 1: Taking it from the wiki.
Loading the most recent change (reliably), or using some kind of API, to find out the stock. 
Isolate the line that says "Current stock:" and then allow the text after it, stopping at "Previous stock:". 
Sending that line via Discord. 
This process has to be done by loading it from the wiki, each time someone sends a command, preferably in real-time.

Alternatively, we could make the bot check for more recent chances. If there is no recent changes, it will just take the old one (`rev1`), storing the old one (`rev1`) until a new revision (`rev2`) appears, then deleting the older one (`rev1`) when a newer one appears.
Basically, we have to find out a way to store these files, delete them, pretty dynamically. 

As is the case, this bot is probably going to receive a lot of traffic, if it is to be used in the Blox Fruits Discord. We need to make sure that the bot will work, and not unexpectedly crash. If it does crash, reboot itself.

**Comment by tort-oise** [comment]
Currently, the bot only stores things statically, and can in no way download data from any of these sources. It is just a plain string of text. This is practically my first big-scale bot, so I would need some severe help about it.

**Comment by tort-oise** [comment]
Update: I've added a commit yesterday that lets me use a .txt file, instead of having to modify stock.js. Pretty significant progress, and still lightweight.
